dev/io.openliberty.org.hibernate.validator.9.0/bnd.overrides

```diff
@@ -0,0 +1,79 @@
+-include= ~../cnf/resources/bnd/rejar.props, \
+          ~../cnf/resources/bnd/app-resources.bnd
+bVersion=1.0
+bSymbolicName=io.openliberty.org.hibernate.validator.9.0
+
+Bundle-Name: Hibernate Validator Engine
+Bundle-SymbolicName: io.openliberty.org.hibernate.validator.9.0
+Bundle-Description: Hibernate's Jakarta Bean Validation 3.1 reference implementation; version=9.0.0.Beta3
+
+# Provide these as resources to applications
+app-resources= \
+  META-INF/services/jakarta.validation.spi.ValidationProvider | \
+  META-INF/validation-mapping-1.0.xsd | \
+  META-INF/validation-configuration-1.0.xsd | \
+  META-INF/validation-mapping-1.1.xsd | \
+  META-INF/validation-configuration-1.1.xsd | \
+  META-INF/validation-mapping-2.0.xsd | \
+  META-INF/validation-configuration-2.0.xsd | \
+  META-INF/validation-mapping-3.0.xsd | \
+  META-INF/validation-configuration-3.0.xsd
+
+Export-Package: \
+  org.hibernate.validator.*;version="9.0.0.Beta3";thread-context=true
+
+Import-Package: \
+  !jakarta.persistence, \
+  jakarta.validation;version="[3.1.0,4.0.0)", \
+  jakarta.validation.constraints;version="[3.1.0,4.0.0)", \
+  jakarta.validation.constraintvalidation;version="[3.1.0,4.0.0)", \
+  jakarta.validation.executable;version="[3.1.0,4.0.0)", \
+  jakarta.validation.groups;version="[3.1.0,4.0.00)", \
+  jakarta.validation.metadata;version="[3.1.0,4.0.0)", \
+  jakarta.validation.spi;version="[3.1.0,4.0.00)", \
+  jakarta.validation.valueextraction;version="[3.1.0,4.0.0)", \
+  javax.script;version=!, \
+  javax.xml.namespace;version=!, \
+  javax.xml.stream;version=!, \
+  javax.xml.stream.events;version=!, \
+  javax.xml.transform;version=!, \
+  javax.xml.transform.stream;version=!, \
+  javax.xml.validation;version=!, \
+  jakarta.el;version="[5.0.0,7.0.0)";resolution:=optional, \
+  org.xml.sax;version=!, \
+  org.jboss.logging;version="[3.1.0,4.0.0)", \
+  com.fasterxml.classmate;version="[1.3,2.0.0)", \
+  com.fasterxml.classmate.members;version="[1.3,2.0.0)", \
+  org.joda.time;version="[2.0.0,3.0.0)";resolution:=optional, \
+  org.jsoup;version="[1.5.2,2.0.0)";resolution:=optional, \
+  org.jsoup.nodes;version="[1.5.2,2.0.0)";resolution:=optional, \
+  org.jsoup.parser;version="[1.5.2,2.0.0)";resolution:=optional, \
+  org.jsoup.safety;version="[1.5.2,2.0.0)";resolution:=optional, \
+  javax.money;version="[1.0.0,2.0.0)";resolution:=optional, \
+  com.thoughtworks.paranamer;version="[2.5.5,3.0.0)";resolution:=optional, \
```

Comment:
Versions on these should be checked, and some may be removable. From what I can see HV no longer depends on `org.jsoup`

https://mvnrepository.com/artifact/org.hibernate.validator/hibernate-validator/9.0.0.Beta3